apps/sel4test-tests/arch/arm/arch/debug.h

```diff
@@ -5,12 +5,21 @@
  */
 #pragma once
 
+#ifdef CONFIG_ARCH_AARCH32
 #define TEST_SOFTWARE_BREAK_ASM() \
     asm volatile( \
         ".global sbreak, post_sbreak\n\t" \
         ".type post_sbreak, function\n\t" \
         "sbreak:\n\t" \
         "bkpt\n\t")
+#else
```

Comment:
Perhaps make it an explicit check for AARCH64, to make it clearer what's happening.

```suggestion
#elif CONFIG_ARCH_AARCH64
```

If you tested this assembly locally I'm happy to merge this PR.

Reply:
Yep, the test passes with the changes I've made in https://github.com/alwin-joshy/seL4/pull/7